### ISSUE TITLE
Not responding when use Chinese language model

### ISSUE THREAD

**Issue by sdaheng** [Funnyguy77/PocketSphinxUnityDemo#10]
I downloaded model from CMUSphnix, and added it to StreamingAssets.
Then I changed configurations, such as lang and keyphrase in inspector and scripts.
When I run the project, it is not responding.
I have no idea why... 
Here are some snippets
<img width = "789" alt= "image" src= "https://user-images.githubusercontent.com/1405754/77528862-9e355a80-6ec9-11ea-8f27-13bca3a97389.png">

<img width= "788" alt= "image" src= "https://user-images.githubusercontent.com/1405754/77528893-ae4d3a00-6ec9-11ea-9c39-026541ab93db.png">

This model seems it should use "SetupDecoderLM()",

The model`s name downloaded from CMUSphnix is not "zh-cn", I renamed it.

Do you have any clue why this happened, and how do I fix it.

My Unity version is 2019.2.21f1
OS version is macOS 10.15.3

Thanks a lot.

**Comment by Funnyguy77** [comment]
Hey there! Thanks for checking out my repo! 

I downloaded the same model for Mandarin Chinese. It looks like you would need to use `SetupDecoderKWS();` for this model. 

You can use the `SetupDecoderLM();` and I encourage you to do so, but you will first need to train your own custom language model before you use the LM search mode. [[Link]](https://cmusphinx.github.io/wiki/tutoriallm/)

Here's what the README for the zh-cn model says:

> One have to train a proper language model for this.

So go ahead and try switching to `SetupDecoderKWS();` and it should spot your keyphrase. But I would encourage you to train a language model if you have the time. Then, you should be able to switch over to using `SetupDecoderLM();`

**Comment by sdaheng** [comment]
Thanks for your reply!
 
I found the problem is the location of mdef, dictionary and other files not in the root directory. I moved files to the zh-cn directory and it never get stuck.

My purpose is to say a command then do something. KWS looks enough for me?

But, I have another problem, d.Hyp() always return NULL. I tried #1 solutions, set -kws_threshold from 1e-1 to 1e-50, but it still NULL... Did I say too short?
